duden/cli.py

```diff
@@ -114,6 +117,8 @@ def parse_args():
 
     parser.add_argument('-V', '--version', action='store_true',
                         help=_('print program version'))
+    parser.add_argument('--phonetic', action='store_true',
+                        help=('displays pronunciation'))
```

Comment:
In this codebase, `_` is a function that localizes strings to various languages.

```suggestion
                        help=_('display pronunciation'))
```

Reply:
(trivial)
Btw. if you know german, spanish or esperanto, please translate the strings
* `'Pronunciation:'`
* `'display pronunciation'`
in the `duden/locale/*.po` files

If you don't, no problem, the localization will default to the default string (english) if not supplied.